src/webhooks/discord/templates/creatorRegistered.ts

```diff
@@ -0,0 +1,40 @@
+import { Prisma } from '@prisma/client';
+import { MessagePayload } from 'discord.js';
+
+export const CREATOR_REGISTERED = (
+  creator: Prisma.CreatorCreateInput,
```

Comment:
Isn't this type misleading? Shouldn't it be the type `Creator` imported from `@prisma/client`?